src/Microsoft.AspNetCore.SignalR.Core/DynamicHubClients.cs

```diff
@@ -15,17 +15,83 @@ public DynamicHubClients(IHubCallerClients clients)
             _clients = clients;
         }
 
+        /// <summary>
+        /// Gets a caller to all hub clients.
```

Comment:
This applies to all of these in all the proxies: How about:

`Gets an object that can be used to invoke methods on all hub clients`

(replacing `all hub clients` as necessary for each of the methods)